src/generator/types.rs

```diff
@@ -16,6 +16,7 @@ impl ExprFolder {
             Expr::Call { .. } => ExprFolder::fold_call(expr, table),
             Expr::VarId { .. } => ExprFolder::fold_var_id(expr, table),
             Expr::Primitive { .. } => ExprFolder::fold_primitive(expr, table),
+            Expr::GetMap { .. } => expr.clone(),
```

Comment:
Couldn't this have things to fold inside it? e.g.

`count.get(target_fn_name)`